Rakefile

```diff
@@ -15,12 +15,4 @@ RSpec::Core::RakeTask.new(:spec) do |t|
   t.pattern = 'spec/ops/**/*_spec.rb'
 end
 
-desc 'Executes the example tests'
-task 'test:examples' do
-  %w(rails_app_sample sinatra).each do |ex|
```

Comment:
These sample apps had already been removed so this Rake task didn't work anymore.